source/partials/practical-details/_venue.slim

```diff
@@ -17,6 +17,6 @@
 
         br
         br
-        | The venue for Mirror Conf will be Auditório Vita, a beautiful piece of architecture. Only 5 minutes away from downtown, this is the perfect place to welcome your creative minds.
+        | The venue for Mirror Conf will be Auditório Vita, a beautiful piece of architecture. Only 5 minutes away from downtown, this is the perfect place to welcome your creative minds. Located at Rua de São Domingos 94B, 4710-435 Braga
```

Comment:
I was actually planning about talking to him about this, didn't notice you had already talked about it. That's why I'm asking